contracts/src/v0.1/mocks/RequestResponseConsumerMock.sol

```diff
@@ -29,7 +35,7 @@ contract RequestResponseConsumerMock is RequestResponseConsumerBase {
         uint64 accId,
         uint32 callbackGasLimit
     ) public onlyOwner returns (uint256 requestId) {
-        bytes32 jobId = keccak256(abi.encodePacked("any-api-int256"));
+        bytes32 jobId = keccak256(abi.encodePacked("Uint256"));
```

Comment:
Let's keep the string with all lowercase as shown at https://orakl-network.gitbook.io/docs/developers-guide/request-response#request-and-response-post-processing

It applies to the request below as well.

Reply:
roger that! let me fix it